modules/stanford_profile_styles/lib/scss/component/node/stanford_opportunity.scss

```diff
@@ -122,6 +142,12 @@
     }
   }
 
+  .su-card {
+    .su-opp-image {
```

Comment:
do you need the additional selector?

Reply:
No I fixed the margin-bottom so I don't need it at all.